### ISSUE TITLE
Will the banned words list be configurable?

### ISSUE THREAD

**Issue by TheBlackProgrammer** [comet-app/server#4]
In https://github.com/comet-app/server/blob/master/src/bannedWords.ts, the banned words list is hardcoded and would require the code to be recompiled to adjust it. Do you plan to add something like a configuration file for it or a way to change it on the website itself if you're an admin?

**Comment by danbeneventano** [comment]
Yes, current plan is to just store them in an environment variable. Had to just throw together a quick hardcoded list because of an emergency situation on the site the other day
